### PR TITLE
feat: add window rule to sample config for ignoring picture-in-picture

### PR DESCRIPTION
**Pull request by AmineDjeghri**

### CODE REVIEW

**Comment by lars-berger** [comment]
Ay thank you 👍 Could you actually add it directly to the [sample config](https://github.com/glazerdesktop/GlazeWM/blob/develop/GlazeWM.App/Resources/sample-config.yaml) instead? It'd be great to just ship this by default imo

**Comment by AmineDjeghri** [comment]
@lars-berger 
Ofc
Do you want me to keep it also in the readme ?

File: GlazeWM.App/Resources/sample-config.yaml [hunk 1]
```diff
@@ -95,6 +95,11 @@ window_rules:
   - command: "set floating"
     match_class_name: "#32770"
 
+# Do not manage Picture-in-picture windows. Default is for english, change the match_title to the appropriate window's name if you use another language. 
+  - command: "ignore"
+    match_title: "[Pp]icture.in.[Pp]icture"
+    match_class_name: "Chrome_WidgetWin_1|MozillaDialogClass"
```
Comment (lars-berger) [inline GlazeWM.App/Resources/sample-config.yaml:101]: Suggested revision for the config comment (mainly for the sake of indentation + cutting of the line length past 90 char):
```yaml
  # Do not manage picture-in-picture windows for browsers. Note that the default is for
  # English; change `match_title` to the window's title if you use another language.
  - command: "ignore"
    match_title: "[Pp]icture.in.[Pp]icture"
    match_class_name: "Chrome_WidgetWin_1|MozillaDialogClass"
```

**Comment by lars-berger** [comment]
@AmineDjeghri Yeah I reckon we can get remove it from the readme. Added a minor comment, otherwise good to go 👍

**Comment by AmineDjeghri** [comment]
@lars-berger i removed it from the readme.

**Pull request merged**